### PR TITLE
Release 2.5.1

### PR DESCRIPTION
**Pull request by YuyaoHu**
Things to consider before you submit the PR:

* [x] Has `CHANGELOG.md` been updated?
* [ ] Are tests passing locally?
* [ ] Are the files formatted correctly?
* [ ] Did you add unit tests if this modifies the Windows code?
* [ ] Did you test your change with either the sample apps that are included in the repository or with a blank app that uses your change?

## Description

Start release 2.5.1

### CODE REVIEW

File: RELEASE.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
-## Release 2.5.0
+## Release 2.5.1
 
 Updated native SDK versions:
-* Android from `2.3.0` to [2.4.1](https://github.com/Microsoft/AppCenter-SDK-Android/releases/tag/2.4.1)
-* iOS from `2.4.0` to [2.5.0](https://github.com/Microsoft/AppCenter-SDK-Apple/releases/tag/2.5.0)
-* UWP from `2.1.0` to [2.5.0](https://github.com/Microsoft/AppCenter-SDK-DotNet/releases/tag/2.5.0)
+* Android from `2.4.1` to [2.5.0](https://github.com/Microsoft/AppCenter-SDK-Android/releases/tag/2.5.0)
```
Comment (guperrot) [inline RELEASE.md]:
```suggestion
* Android from `2.3.0` to [2.5.0](https://github.com/Microsoft/AppCenter-SDK-Android/releases/tag/2.5.0)
```

File: RELEASE.md [hunk 2]
```diff
@@ -1,9 +1,9 @@
...
+* iOS from `2.5.0` to [2.5.1](https://github.com/Microsoft/AppCenter-SDK-Apple/releases/tag/2.5.1)
+* UWP from `2.5.0` to [2.6.1](https://github.com/Microsoft/AppCenter-SDK-DotNet/releases/tag/2.6.1)
```
Comment (guperrot) [inline RELEASE.md]:
```suggestion
* UWP from `2.1.0` to [2.6.1](https://github.com/Microsoft/AppCenter-SDK-DotNet/releases/tag/2.6.1)
```

File: RELEASE.md [hunk 3]
```diff
@@ -1,9 +1,9 @@
-## Release 2.5.0
+## Release 2.5.1
 
 Updated native SDK versions:
-* Android from `2.3.0` to [2.4.1](https://github.com/Microsoft/AppCenter-SDK-Android/releases/tag/2.4.1)
-* iOS from `2.4.0` to [2.5.0](https://github.com/Microsoft/AppCenter-SDK-Apple/releases/tag/2.5.0)
-* UWP from `2.1.0` to [2.5.0](https://github.com/Microsoft/AppCenter-SDK-DotNet/releases/tag/2.5.0)
+* Android from `2.4.1` to [2.5.0](https://github.com/Microsoft/AppCenter-SDK-Android/releases/tag/2.5.0)
+* iOS from `2.5.0` to [2.5.1](https://github.com/Microsoft/AppCenter-SDK-Apple/releases/tag/2.5.1)
```
Comment (guperrot) [inline RELEASE.md]:
```suggestion
* iOS from `2.4.0` to [2.5.1](https://github.com/Microsoft/AppCenter-SDK-Apple/releases/tag/2.5.1)
```

File: RELEASE.md (hunk 1 above)
Comment (guperrot) [reply RELEASE.md]: Because needs to be from last release we actually shipped.

Comment (YuyaoHu) [reply RELEASE.md]: I've made changes, please check out. @guperrot

File: RELEASE.md (hunk 2 above)
Comment (YuyaoHu) [reply RELEASE.md]: I've made changes, please check out

**Review by Cloudwu1 — APPROVED** [review]

**Pull request merged**